src/scripts/main.js

```diff
@@ -1,8 +1,17 @@
 'use strict';
 
 const people = require('./lib/people');
+const dashTable = document.querySelector('.dashboard');
```

Comment:
Consider using 'const dashboardTable' instead of 'const dashTable' for better clarity.